xstream/src/test/com/thoughtworks/xstream/io/naming/StaticNameCoderTest.java

```diff
@@ -0,0 +1,64 @@
+package com.thoughtworks.xstream.io.naming;
+
+import com.thoughtworks.xstream.io.naming.StaticNameCoder;
+import junit.framework.TestCase;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class StaticNameCoderTest extends TestCase {
+
+  public void testDecodeAttribute() {
+    final StaticNameCoder staticNameCoder = new StaticNameCoder(new HashMap<String, String>(), new HashMap<String, String>());
```

Comment:
break lines after 80 chars

Reply:
Renamed stuff so that no line is over 80 chars.